.circleci/config.yml

```diff
@@ -119,23 +119,35 @@ jobs:
       - run:
           name: Install interledgerjs/five-bells-ledger-api-tests
           command: npm install github:interledgerjs/five-bells-ledger-api-tests
+      - run:
+          name: Build dependencies
+          command: apk add --no-cache -t build-dependencies make gcc g python libtool autoconf automake
```

Comment:
this is supposed to be: 
apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake

you didnt put "g++"